src/Slices/Status/UI/Page.test.tsx

```diff
@@ -100,6 +95,11 @@ function setup(useMockArchiveHelper = false) {
     archiveHelper,
   };
 }
+const server = setupServer();
+
+beforeAll(() => server.listen());
```

Comment:
here too, to avoid leaks and flakes, this should only work within a describe section.